ts/screens/services/ServiceDetailsScreen.tsx

```diff
@@ -60,41 +58,8 @@ type Props = ReturnType<typeof mapStateToProps> &
   OwnProps;
 
 const styles = StyleSheet.create({
-  infoHeader: {
-    marginTop: customVariables.spacerLargeHeight
-  },
-  infoItem: {
-    flexDirection: "column",
-    marginTop: customVariables.spacerHeight
-  },
-  badgeLogo: {
-    width: 150,
-    height: 50,
-    alignSelf: "flex-start"
-  },
-  switchRow: {
-    justifyContent: "space-between",
-    paddingVertical: 12
-  },
-  otherSwitchRow: {
-    borderTopColor: customVariables.itemSeparator,
-    borderTopWidth: 1 / 3
-  },
   flexRow: {
     flexDirection: "row"
```

Comment:
I guess we may remove even this style definition using the globally defined `IOStyles.row`, what do you think?

Reply:
ref: https://github.com/pagopa/io-app/blob/3e2faceb3505e90678040d674c1bcaeafb84b866/ts/components/core/variables/IOStyles.ts#L18